docs/community_features.md

```diff
@@ -15,7 +15,9 @@ Here is a list of general improvements that have been made, ordered from newest
 - ([#17]) Increase the resolution of "patch cables" between mod sources and destinations.
 
 #### 3.2 - MPE
-- ([#29]) Bugfix to respect MPE zones in kit rows. In the official firmware kit rows with midi learned to a channel would be triggered by an MPE zone which uses that channel. With this change they respect zones in the same way as synth and midi clips.
+- ([#29]) Bugfix to respect MPE zones in kit rows. In the official firmware kit rows with midi learned to a channel would be triggered by an MPE zone which uses that channel. With this change they respect zones in the same way as synth and midi clips. ([512]) adds further fixes related to channels 0 and 15 always getting received as MPE.
```

Comment:
`512` -> `#512`, and  you need to add an entry at the end of the document to the list of links